tests/src/test/java/alluxio/master/MasterFaultToleranceIntegrationTest.java

```diff
@@ -58,9 +56,9 @@ public final void before() throws Exception {
     // TODO(gpang): Implement multi-master cluster as a resource.
     mMultiMasterLocalAlluxioCluster =
         new MultiMasterLocalAlluxioCluster(MASTERS);
+    mMultiMasterLocalAlluxioCluster.initConfiguration();
```

Comment:
`initConfiguration()` sets most Alluxio configuration to default values, so it needs to happen before `Configuration.set`, or it will override the values we want.